alfalfa_worker/lib/tagutils.py

```diff
@@ -1,21 +1,8 @@
 import json
 import uuid
 
-
-# replace_ids.replace_siteid(upload_id, points_jsonpath, mapping_jsonpath)
-def replace_site_id(uploadid, points_jsonpath, mapping_jsonpath):
-    # step-1: find the siteid from jsonfile
-    with open(points_jsonpath, 'r') as jsonfile:
-        data = json.load(jsonfile)
-        for x in data:
-            for y in x.keys():
-                if 'site' == y:
-                    x['id']
-                else:
-                    pass
-
-    # step-2: replace the siteid with uploadid
-    for x in data:
+def replace_site_id(uploadid, points_json):
```

Comment:
@kbenne  same: we need to either re-add 3rd argument or update calls to this function.